requirements/pip.txt

```diff
@@ -15,9 +15,9 @@ djangorestframework==3.12.4
 Sphinx==4.2.0
 
 # Filtering for the REST API
-django-filter==2.4.0
+django-filter==21.1
 
-drf-flex-fields==0.9.3
+drf-flex-fields==0.9.4
```

Comment:
This one looks suspicious :thinking:... https://github.com/rsinger86/drf-flex-fields/issues/85. I'll check the code from pypi later

```suggestion
drf-flex-fields==0.9.3
```

Reply:
it was just a formatting change

```
$ diff -r drf-flex-fields drf-flex-fields-0.9.4 
diff --color -r drf-flex-fields/rest_flex_fields/serializers.py drf-flex-fields-0.9.4/rest_flex_fields/serializers.py
142,145c142
<         self,
<         omit_fields: List[str],
<         sparse_fields: List[str],
<         next_level_omits: List[str],
---
>         self, omit_fields: List[str], sparse_fields: List[str], next_level_omits: List[str]
Only in drf-flex-fields-0.9.4: setup.cfg
diff --color -r drf-flex-fields/setup.py drf-flex-fields-0.9.4/setup.py
27c27
<     version="0.9.3",
---
>     version="0.9.4",
```